app/models/comment.rb

```diff
@@ -1,4 +1,5 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
```

Comment:
- [ ] According to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) you should have comment.rb belongs to the author and it should have classname 'User'.